mandatory/4-tax.js

```diff
@@ -5,7 +5,11 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(price) {
+  return (price * 20) / 100;
```

Comment:
This is nice. 
First, you are not returning to the correct format with the £ symbol